aray/array-exercises.js

```diff
@@ -0,0 +1,101 @@
+// exercises
+
+var a = [1, 2, 5, 8, 6, 9, 3];
+const odd = [];
+
+a.forEach((number) => {
+  if (number % 2 === 1) {
+    odd.push(number);
+  }
+});
+console.log(odd);
+
+// ex2
+
+const b = [1, 5, 8, 7, 2];
+b.forEach((num, index) => {
+  b[index] = num + 1;
+});
+console.log(b);
+
+//  ex 3
+
+const words = [
+  "spray",
+  "limit",
+  "elite",
+  "exuberant",
+  "destruction",
+  "present",
+];
+const lastItem = words[words.length - 1];
+console.log(lastItem);
+
+// ex4
+
+const array = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10];
+const odd2 = [];
+const oddNumbers = array.map((number2) => {
+  if (number2 % 2 === 1) {
+    odd2.push(number2);
```

Comment:
You don't need to use map here. There's a specific array function that you can use to have only the odd values inside another array. Which function do you think you need to use here?